`import '@jscutlery/cypress-harness/support';` causes Webpack Compilation Error with Angular 17

When setting up Cypress Harness according to the documentation in a project with Angular 17, E2E tests fail to run with the following error:
```
  Running:  app.cy.ts                                                                       (1 of 1)
Oops...we found an error preparing this test file:

  > src/support/e2e.ts

The error was:

Error: Webpack Compilation Error
Module not found: Error: Package path ./dist/zone is not exported from package...
...
```
We noticed that removing `import '@jscutlery/cypress-harness/support';` from `cypress/support/e2e.ts` allows the tests to prepare and run again.

Furthermore (maybe a separate and unrelated issue), we are unable to do `import { getPreprocessorConfig } from '@jscutlery/cypress-harness/preprocessor-config';` because of error `Cannot find module '@jscutlery/cypress-harness/preprocessor-config' or its corresponding type declarations.ts(2307)`.

Issue reproducible [here](https://github.com/abaran30/nx-angular-playground/tree/cypress-13-harness).

Steps to reproduce:
- Checkout repository
- `git checkout cypress-13-harness`
- `npm ci` (if fails, try with `--legacy-peer-deps`)
- Attempt `npx nx run my-app-a-e2e:e2e` - will fail with the aforementioned error

--- a/apps/sandbox/src/polyfills.ts
+++ b/apps/sandbox/src/polyfills.ts
@@ -55,7 +55,7 @@
 /***************************************************************************************************
  * Zone JS is required by default for Angular itself.
  */
-import 'zone.js/dist/zone'; // Included with Angular CLI.
+import 'zone.js'; // Included with Angular CLI.
 
 /***************************************************************************************************
  * APPLICATION IMPORTS

--- a/packages/cypress-harness/src/support.ts
+++ b/packages/cypress-harness/src/support.ts
@@ -6,10 +6,10 @@ import 'cypress-pipe';
  * in `@angular/core/testing`.
  * Cf. https://github.com/jscutlery/devkit/issues/2
  */
-import 'zone.js/dist/zone';
+import 'zone.js';
 
 /* @hack fixes "Mocha has already been patched with Zone" error. */
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
 (globalThis as any)['Mocha']['__zone_patch__'] = false;
 
-import 'zone.js/dist/zone-testing';
+import 'zone.js/testing';

